lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -37,7 +37,7 @@ export default TestModule.extend({
       this.setupSteps.push(this.setupComponentUnitTest);
     } else {
       this.callbacks.subject = function() {
-        throw new Error("component integration tests do not support `subject()`.");
+        throw new Error("component integration tests do not support `subject()`. Instead, render the component as if it were HTML: `this.render('<my-component foo=true>');`. For more information, read: https://github.com/switchfly/ember-test-helpers/pull/38");
```

Comment:
@rwjblue I'm not sure about the language on this, or even if the suggestion is correct.

Once we nail down the language of this warning, I'd gladly add to the README's explanation on `integration: true` mode.